scripts/request-vrf-direct.ts

```diff
@@ -1,16 +1,17 @@
 import { ethers, getNamedAccounts } from 'hardhat'
+import { getKeyHash } from './utils'
 
 async function main() {
   const vrfConsumer = await ethers.getContract('VRFConsumer')
 
-  const keyHash = '0xd9af33106d664a53cb9946df5cd81a30695f5b72224ee64e798b278af812779c'
+  const keyHash = getKeyHash()
   const callbackGasLimit = 500_000
   const numWords = 1
 
   const { deployer } = await getNamedAccounts()
   const txReceipt = await (
     await vrfConsumer.requestRandomWordsDirect(keyHash, callbackGasLimit, numWords, deployer, {
-      value: ethers.utils.parseEther('1.0')
+      value: ethers.utils.parseEther('2.5')
```

Comment:
Wouldn't it be better to make call for estimating the service fee so we don't have to update this amount if the price changes?

Reply:
@martinkersner What you suggested is very good. 

Now, I have added your suggestion, where we get estimatedServiceFee while calling direct request. Could you please check it out?